src/content/en/tools/chrome-devtools/debug/remote-debugging/remote-debugging.markdown

```diff
@@ -1,168 +1,165 @@
+any intermediate hubs. 
+If this is your first time connecting this Android device to this development
+machine, you should see an unknown device that is pending authorization on
+your Inspect Devices dialog.
 
-On your desktop Chrome browser, navigate to **chrome://inspect**. Confirm that **Discover USB devices** is checked:
+![pending authorization](imgs/pending-authorization.png)
 
-![Discover USB Devices in chrome://inspect](imgs/chrome-discover-usb.png)
+If this is the case, then you need to open the **Allow USB debugging** prompt
+on your Android device and grant the permissions. 
 
-**Tip**: You can also get to **chrome://inspect** by selecting **Chrome menu > More tools > Inspect Devices**.
+**Tip**: If you have encounter any issues during the discovery process, you 
+can start the process anew by going back to **Developer options** and tapping 
+**Revoke USB debugging authorizations**. 
 
-On your device, an alert prompts you to allow USB debugging from your computer. Tap **OK**.
+After allowing USB debugging, you should see your device in the Inspect Devices
+dialog. 
 
-![USB debugging permission alert](imgs/rsa-fingerprint.png)
+![inspect devices dialog with connection](imgs/connected-inspect-devices.png)
 
-**Tip**: To skip this alert in the future, check **Always allow from this computer**.
+## Debug content on Android device from development machine
 
-The message **USB debugging connected** displays in the device's notification drawer.
+From the Inspect Devices dialog, select your device from the menu on the left.
 
-**Note**: During remote debugging, Chrome prevents your device’s screen from going to sleep. This feature is useful for debugging, but is also less secure. So be sure to keep an eye on your device!
+![inspecting a device](imgs/selected-inspect-device.png)
 
-If you have problems finding your device on the **chrome://inspect page**, see the [Troubleshooting](#troubleshooting) section.
+From here you can see a variety of information about your connected Andoid 
+device:
 
-### Ready to debug your site in a browser tab!
+* At the top you can see the model name of your Android device followed by 
+  its serial number (for example, `Nexus 5 #08ae8c2700f43a61`).
+* If one or more Chrome tabs are open, then you'll see a **Chrome** heading
+  followed by the version number of Chrome that's running (for example,
+  `Chrome (49.0.2623.105)`). If no Chrome tabs are open, you won't see a 
+  Chrome heading. 
+* Underneath the **Chrome** heading, each open tab gets its own heading. 
+  You can interact with that tab from this section. 
+* If there are any running apps using [WebView][webview], you'll see a heading 
+  for each app. 
 
-On your computer, the **chrome://inspect** page displays every connected device, along with its open tabs and debug-enabled [WebViews](/web/tools/chrome-devtools/debug/remote-debugging/webviews).
+To open a new Chrome tab, enter a URL in the textfield under the Chrome heading
```

Comment:
Interesting! Tricky case actually, since a lot of people pronounce it like
"earl". anyway, both work for me :)
On Apr 12, 2016 10:43 AM, "Kayce Basques" <notifications@github.com> wrote:

> In
> src/content/en/tools/chrome-devtools/debug/remote-debugging/remote-debugging.markdown
> <https://github.com/google/WebFundamentals/pull/2792#discussion_r59420134>
> :
>
> >
> > -On your computer, the **chrome://inspect** page displays every connected device, along with its open tabs and debug-enabled [WebViews](/web/tools/chrome-devtools/debug/remote-debugging/webviews).
> > +To open a new Chrome tab, enter a URL in the textfield under the Chrome heading
>
>
> http://itknowledgeexchange.techtarget.com/writing-for-business/which-is-correct-a-url-or-an-url/
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/WebFundamentals/pull/2792/files/143cb6db5425a4748a0916e83a9cce54e6680df7#r59420134>
>